persist/storer.go

```diff
@@ -89,7 +91,8 @@ func DefaultPath(integrationName string) string {
 func NewInMemoryStore() Storer {
 	return &inMemoryStore{
 		cachedData: make(map[string]jsonEntry),
-		Data:       make(map[string][]byte),
+		data:       make(map[string][]byte),
```

Comment:
I think this field has to be public (start with uppercase) because otherwise it won't be serialized into disk.